ci-operator/step-registry/openshift/e2e/test/staging/openshift-e2e-test-staging-ref.yaml

```diff
@@ -0,0 +1,62 @@
+ref:
```

Comment:
maybe i missed it, what is “staging” and why is the normal e2e test stage insufficient?

Reply:
Staging required due to es.redhat.com/browse/DPTP-2114, so that rehearsals would pick disruptive test.

I'll re-submit ref change separately before proceeding